src/Downloader.ts

```diff
@@ -151,6 +156,14 @@ class Downloader {
     }
   }
 
+  public isImageUrl = (url: string): boolean => {
+    return IMAGE_REGEX.exec(path.extname(url).toLowerCase()) ? true : false;
```

Comment:
I am not sure whether i understood the comment correctly. Why exactly we want to change toLowerCase() here? I used it because it will convert all the uppercases to lowercase and then in regex we do not have to put different checks for uppercases values like JPG, GIF etc.

Reply:
This complexify the code without good reason and this is less efficient than the proper regex. See https://www.w3schools.com/jsref/jsref_regexp_i.asp

Reply:
ok, got it.